pkg/adapter/canadapter/sequence.go

```diff
@@ -25,7 +25,8 @@ const MaxFrameNum = 31
 // Sequence frames can be received in any order.
 // we track the time started to allow for releasing incomplete sequences (TBD).
 type sequence struct {
-	started  time.Time
+	started  int64
```

Comment:
are these started/recent times used for anything or just debugging?

Reply:
"started" was added initially to deal with incomplete sequences but never used. I've deleted it and the debugging timestamp "recent" and fixed the comment.